rust/arrow/src/ipc/writer.rs

```diff
@@ -984,7 +984,6 @@ mod tests {
             "generated_dictionary",
             // "generated_duplicate_fieldnames",
             "generated_interval",
-            "generated_large_batch",
```

Comment:
@nevi-me I don't remember seeing this in the original PR -- was this change intended ?

Reply:
NM I see https://github.com/apache/arrow/pull/9587 now